core/include/scipp/core/variable.h

```diff
@@ -1062,8 +1062,19 @@ SCIPP_CORE_EXPORT VariableView atan(const VariableConstView &var,
 SCIPP_CORE_EXPORT VariableView nan_to_num(const VariableConstView &var,
                                           const VariableConstView &replacement,
                                           const VariableView &out);
+SCIPP_CORE_EXPORT VariableView
+pos_inf_to_num(const VariableConstView &var,
+               const VariableConstView &replacement, const VariableView &out);
+SCIPP_CORE_EXPORT VariableView
+neg_inf_to_num(const VariableConstView &var,
```

Comment:
Do these names mirror anything in Numpy? If not is it worth dropping the `pos`/`neg` abbreviations for the sake of readability?

Reply:
The corresponding numpy arguments to `nan_to_num` are `posinf`, `neginf` you would prefer `positive_inf_to_num`? I would be ok doing that if you think it would improve readability.

Reply:
If it's close to how Numpy names things we should keep that, but my preference is to avoid unnecessary abbreviations.